### ISSUE TITLE
TypeScript package for SCIP

### ISSUE THREAD

**Issue by NicholasLYang** [sourcegraph/scip#253]
Hey, love the work on SCIP. I've been playing around with it in Rust, I was wondering if the TypeScript bindings are published on npm? If not, is that possible?

**Comment by varungandhi-src** [comment]
No, they haven't been published to npm. Have you tried using it as a git-based dependency for now? We can look into this.

**Comment by NicholasLYang** [comment]
Sounds good, I can do that in the meanwhile. Thanks for responding!

**Comment by connor4312** [comment]
It'd be nice to get this published. npm [doesn't](https://github.com/npm/npm/issues/2974) seem to have a way to install modules from git subfolders, so making it work is quite a pain.
